### PR TITLE
Fix blocking in xprof

### PR DESCRIPTION
**Pull request by TApplencourt**
Stupid mistake on my side...

### CODE REVIEW

File: xprof/xprof.rb.in [hunk 1]
```diff
@@ -366,10 +366,11 @@ def env_tracers
     h["LTTNG_UST_#{name.upcase}_PROFILE"] = 1 if OPTIONS[:profile]
     h["LTTNG_UST_#{name.upcase}_VERBOSE"] = 1 if LOGGER.level <= Logger::DEBUG
   end
-
+  # Usr Apps need to be launched with this to support blocking.
+  h['LTTNG_UST_ALLOW_BLOCKING'] = 1
   # Customization
   h['LTTNG_UST_ZE_PARANOID_DRIFT'] = 1 if OPTIONS[:'backend-names'].include?('ze') && OPTIONS[:profile]
-
+
```
Comment (TApplencourt) [inline xprof/xprof.rb.in:373]:
```suggestion

```

**Comment by Kerilk** [comment]
Where is the commit that removed this?

**Comment by Kerilk** [comment]
Broken since 7d445367b87032e5471a1d75b86b1ea9c841d4e7

**Review by Kerilk — APPROVED** [review]

**Comment by TApplencourt** [comment]
7d445367b87032e5471a1d75b86b1ea9c841d4e7 yes, during the ruby rewrite.

**Comment by TApplencourt** [comment]
Even longer `https://github.com/argonne-lcf/THAPI/blob/eae4d42556c2c3d439e287236125380bba2589ba/xprof/xprof.sh.erb.in` inside this log doesn't have it neither

**Comment by Kerilk** [comment]
No, it was there:
https://github.com/argonne-lcf/THAPI/blob/eae4d42556c2c3d439e287236125380bba2589ba/xprof/xprof.sh.erb.in#L472

**Pull request merged**